app/service/rest.py

```diff
@@ -509,13 +510,22 @@ def get_all_notifications_for_service(service_id):
 @service_blueprint.route("/<uuid:service_id>/notifications/count", methods=["GET"])
 def count_notifications_for_service(service_id):
     data = notifications_filter_schema.load(request.args)
-
+    limit_days = data.get("limit_days")
     multidict = MultiDict(data)
 
+    template_types = multidict.getlist("template_type")
+
+    # set default values for template_types and limit_days
+    if limit_days is None or limit_days == 0:
```

Comment:
cos `0` is falsey, we can just streamline this if statement like we do the template_types below
```suggestion
    if not limit_days:
```

Reply:
https://github.com/alphagov/notifications-api/pull/4160/commits/bc6b677fe2132c9506a042caa0b838ee80b63df9